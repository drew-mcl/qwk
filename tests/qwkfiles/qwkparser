# Variables
CC = gcc
CFLAGS = -Wall -g

# Dependencies
all: main.o util.o

# Subtitution1
main.o:
	$(CC) $(CFLAGS) -c main.c

# Subtitution2
util.o:
	$(CC) $(CFLAGS) -c util.c

# Basic
clean:
	rm -f *.o main
