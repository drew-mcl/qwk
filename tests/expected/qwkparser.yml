variables:
  CC: gcc
  CFLAGS: -Wall -g
targets:
  - all
  - main.o
  - util.o
  - clean
